lib/pf/web/externalportal.pm

```diff
@@ -34,6 +35,12 @@ use pf::web::constants;
 use pf::web::util;
 use pf::constants;
 
+# Some vendors don't support some charatcters in their redirect URL
+# This here below allows to map some URLs to a specific switch module
+Readonly our $SWITCH_REWRITE_MAP => {
+    'RuckusSmartZone' => 'Ruckus::SmartZone',
```

Comment:
Shouldn't be the opposite? It looks to me we want to rewrite Ruckus::SmartZone to RuckusSmartZone.